client/ayon_core/hosts/max/plugins/publish/validate_extended_viewport.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from ayon_core.pipeline import PublishValidationError
+from pymxs import runtime as rt
+
+
+class ValidateExtendedViewport(pyblish.api.InstancePlugin):
+    """Validate if the first viewport is an extended viewport."""
+
+    order = pyblish.api.ValidatorOrder
+    families = ["review"]
+    hosts = ["max"]
+    label = "Validate Extended Viewport"
+
+    def process(self, instance):
```

Comment:
```suggestion
    def process(self, context):
```

Reply:
we can use context plugin instead for this case as we dont need to grab any instance data for the validator

Reply:
Thank you. I have done that :)